### ISSUE TITLE
Results differents between IOS and Java version

### ISSUE THREAD

**Issue by gelopfalcon** [dropbox/zxcvbn#226]
Hi what's going on? I've implemented zxcvbn in android and IOS, but the results are differents. For example the strength for the Str1p3s$$ password in IOS t is fair but in Java is strong.
In addition the property strength. crackTimeSeconds isn't in Java but if in IOS.

I'm having some problems with it, in addition, I've used the JS version and the result is "Fair". Could someone say me because the results are differents between platforms? Is it inconcistent?

**Comment by AlexandruSimandi** [comment]
maybe it's because of encoding

**Comment by guillaumeparis2000** [comment]
Same between javascript and Go version. 🤷‍♂️

**Comment by Tostino** [comment]
Which Java version are you using? For nbvcxz here is what I get...

![image](https://user-images.githubusercontent.com/8227578/41547586-953c3f80-72ee-11e8-88bb-9f0dbd7e9214.png)


There will be differences between the implementations. With nbvcxz, I did not agree with how some of the algorithms were implemented, and I also expanded them to cover the limitations I noticed with the original zxcvbn implementation.

I'm sure other ports have done similar, or possibly have just made mistakes in their implementations.
